src/components/layouts/Footer.vue

```diff
@@ -23,6 +23,8 @@
               <a href="https://www.rsk.co/terms-conditions" target="_blank">
                 Terms & Conditions
               </a>
+              <a href="#">Api Version: {{obtainApiVersion}}</a>
```

Comment:
this link should point to the api tag

Reply:
- will send to https://github.com/rsksmart/2wp-api/releases/tag/v${this.apiVersion}